mxcube3/ui/components/SampleView/MotorInput.js

```diff
@@ -65,6 +66,7 @@ export default class MotorInput extends React.Component {
                 step={step}
                 defaultValue={valueCropped}
                 name={motorName}
+                disabled={this.props.state === 4 || this.props.state === 3 || this.props.state <= 1}
```

Comment:
I did this originally but it does not work, the reason is that if you disable OnMove the cursor moves away from the input box and the user needs to reselect it for additional movement